### PR TITLE
Add Change Log file

### PR DESCRIPTION
**Pull request by JeanFred**
This follows the keep-a-changelog standard,
using `github-changelog-generator` as a base.

### CODE REVIEW

**Comment by JeanFred** [comment]
The base was done using:
```
docker run -v $(pwd):/tmp -ti --entrypoint github_changelog_generator rubygem/github_changelog_generator --user cabotapp --project cabot-alert-email --pr-label '' --no-author 
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+- Update documentation with references to email plugin, not hipchat [\#1](https://github.com/cabotapp/cabot-alert-email/pull/1)
+
+## [1.3.0](https://github.com/cabotapp/cabot-alert-email/tree/1.3.0) (2015-02-13)
+[Full Changelog](https://github.com/cabotapp/cabot-alert-email/compare/1.0.1...1.3.0)
+
+## [1.0.1](https://github.com/cabotapp/cabot-alert-email/tree/1.0.1) (2015-02-13)
+[Full Changelog](https://github.com/cabotapp/cabot-alert-email/compare/0.2...1.0.1)
+
+## [0.2](https://github.com/cabotapp/cabot-alert-email/tree/0.2) (2015-02-05)
```
Comment (frankh) [inline CHANGELOG.md]: ordering's a bit screwed up here

**Review by frankh — CHANGES_REQUESTED** [review]
Fix change log ordering between 1.2, 0.2 and 1.0.1

**Comment by JeanFred** [comment]
Good catch, fixed.

**Review by frankh — APPROVED** [review]

**Pull request merged**